rules-reviewed/jboss-eap/websphere/websphere-mq.windup.xml

```diff
@@ -0,0 +1,846 @@
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="websphere" versionRange="(0,)"/>
+        <sourceTechnology id="websphere-mq" versionRange="(0,)"/>
+        <targetTechnology id="eap" versionRange="[6,)"/>
+        <targetTechnology id="jms" versionRange="[6,)"/>
+        <tag>reviewed-2015-06-09</tag>
+        <tag>messaging</tag>
+        <tag>mq</tag>
+        <tag>mqe</tag>
+        <tag>ibm</tag>
+        <tag>jms</tag>
+    </metadata>
+
+    <rules>
+        <!-- MQe*Adapter -->
+        <rule id="websphere-mq-01000">
+            <when>
+                <javaclass references="com.ibm.mqe.adapters.MQe{*}Adapter">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe JMS adapter" effort="4">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/adapters/package-summary.html"/>
+                </classification>
+                <hint>
+                    <message>
+                        WebSphere MQe JMS adapter can be [deployed to JBoss EAP 6](https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Administration_and_Configuration_Guide/Deploy_the_WebSphere_MQ_Resource_Adapter.html).
+                    </message>
+                    <link description="JBoss EAP 6 Administration and Configuration Guide: Deploy the WebSphere MQ Resource Adapter"
+                          href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Administration_and_Configuration_Guide/Deploy_the_WebSphere_MQ_Resource_Adapter.html"/>
+                </hint>
+            </perform>
+        </rule>
+
+        <!-- MQe*AdminMsg -->
+        <rule id="websphere-mq-02000">
+            <when>
+                <javaclass references="com.ibm.mqe.administration.MQe{*}AdminMsg">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification title="WebSphere MQ JMS administration API" description="Provides classes used to administer and monitor a Websphere MQ Everyplace queue manager." effort="4">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/adapters/package-summary.html"/>
+                </classification>
+                <hint>
+                    <message>In JBoss EAP 6, the JMS destinations can be controlled with JBoss CLI.</message>
```

Comment:
Okay, that's good, but we need an explanation of the problem scenario that was found.